src/ui/app_page.rs

```diff
@@ -173,7 +173,7 @@ impl IPresenter for ApplicationsPage {
         area: &ratatui::prelude::Rect,
         frame: &mut ratatui::Frame<'_>,
         model: &std::rc::Rc<Model>,
-        focused: bool,
+        _focused: bool,
```

Comment:
why keep it?

Reply:
because it is used in other implementation of this trait but not in this one